flair/nn/decoder.py

```diff
@@ -121,3 +123,49 @@ def forward(self, embedded):
         scores = -distance
 
         return scores
+
+
+class LabelVerbalizerDecoder(torch.nn.Module):
+    def __init__(self, label_encoder, label_dictionary: Dictionary, decoding: str = "dot-product"):
```

Comment:
DocumentEmbeddings raise ImportError, looks like circular dependency.
Embeddings as used in nn/model.py instead of DocumentEmbeddings works. If we do not want to take Embeddings, I can open up new issue to inspect this.

```ImportError while loading conftest '/Users/jgolde/PycharmProjects/flair/tests/conftest.py'.
tests/conftest.py:6: in <module>
    import flair
flair/__init__.py:28: in <module>
    from . import (  # noqa: E402 import after setting device
flair/models/__init__.py:1: in <module>
    from .clustering import ClusteringModel
flair/models/clustering.py:14: in <module>
    from flair.embeddings import DocumentEmbeddings
flair/embeddings/__init__.py:13: in <module>
    from .document import (
flair/embeddings/document.py:21: in <module>
    from flair.nn import LockedDropout, WordDropout
flair/nn/__init__.py:1: in <module>
    from .decoder import LabelVerbalizerDecoder, PrototypicalDecoder
flair/nn/decoder.py:15: in <module>
    from flair.embeddings import DocumentEmbeddings
E   ImportError: cannot import name 'DocumentEmbeddings' from 'flair.embeddings' (/Users/jgolde/PycharmProjects/flair/flair/embeddings/__init__.py)```